testutils/memory.go

```diff
@@ -0,0 +1,123 @@
+package testutils
+
+import (
+	"github.com/src-d/metadata-retrieval/github/graphql"
+
+	"gopkg.in/src-d/go-log.v1"
+)
+
+// Memory implements the storer interface
+type Memory struct {
+	Organization *graphql.Organization
+	Repository   *graphql.RepositoryFields
+	Topics       []string
+	Users        []*graphql.UserExtended
+	PRs          []*graphql.PullRequest
+	PRComments   []*graphql.IssueComment
+}
+
+// SaveOrganization stores an organization in memory,
+// it also initializes the list of users
+func (s *Memory) SaveOrganization(organization *graphql.Organization) error {
+	log.Infof("organization data fetched for %s\n", organization.Login)
+	s.Organization = organization
+	// Initialize users to 0 for each repo
+	s.Users = make([]*graphql.UserExtended, 0)
+	return nil
+}
+
+// SaveUser appends a user to the user list in memory
+func (s *Memory) SaveUser(user *graphql.UserExtended) error {
+	log.Infof("user data fetched for %s\n", user.Login)
+	s.Users = append(s.Users, user)
+	return nil
+}
+
+// SaveRepository stores a repository and its topics in memory and
+// initializes PRs and PR comments
+func (s *Memory) SaveRepository(repository *graphql.RepositoryFields, topics []string) error {
+	log.Infof("repository data fetched for %s/%s\n", repository.Owner.Login, repository.Name)
+	s.Repository = repository
+	s.Topics = topics
+	// Initialize prs and comments to 0 for each repo
+	s.PRs = make([]*graphql.PullRequest, 0)
+	s.PRComments = make([]*graphql.IssueComment, 0)
+	return nil
+}
+
+// TODO(kyrcha): add memory in noop methods as the tests expand
+
+// SaveIssue noop
+func (s *Memory) SaveIssue(repositoryOwner, repositoryName string, issue *graphql.Issue, assignees []string, labels []string) error {
+	log.Infof("issue data fetched for #%v %s\n", issue.Number, issue.Title)
+	return nil
+}
+
+// SaveIssueComment noop
+func (s *Memory) SaveIssueComment(repositoryOwner, repositoryName string, issueNumber int, comment *graphql.IssueComment) error {
+	log.Infof("  issue comment data fetched by %s at %v: %q\n", comment.Author.Login, comment.CreatedAt, trim(comment.Body))
```

Comment:
why do we need these spaces?

Reply:
I just replicated the [stdout](https://github.com/src-d/metadata-retrieval/blob/master/github/store/stdout.go) store where spaces are used for identing hierarchical structures (Issue > Issue Comments etc.)

Reply:
will it make sense to add `\t` instead of spaces, wdyt?

Reply:
Yes, will add them to be more elegant